python/ray/data/_internal/execution/operators/map_operator.py

```diff
@@ -188,10 +190,27 @@ def add_input(self, refs: RefBundle, input_index: int):
             bundle = self._block_ref_bundler.get_next_bundle()
             self._add_bundled_input(bundle)
 
-    def _get_runtime_ray_remote_args(self) -> Dict[str, Any]:
+    def _get_runtime_ray_remote_args(
+        self, input_bundle: Optional[RefBundle] = None
+    ) -> Dict[str, Any]:
+        ray_remote_args = copy.deepcopy(self._ray_remote_args)
+        if "scheduling_strategy" not in ray_remote_args:
```

Comment:
It would be nice to add a comment here explaining the high-level scheduling policy.